Makefile

```diff
@@ -50,11 +50,7 @@ path_run: path_build check_path_config ## Run the path binary as a standalone bi
 
 .PHONY: path_up
 path_up: check_path_config dev_up ## Brings up local Tilt development environment which includes PATH and all related dependencies (using kind cluster)
-	tilt up # MODE=path_with_auth is the default
-
-.PHONY: path_up_standalone
```

Comment:
I'm still seeing references to this `path_up_standalone` and personally don't know how to run non-envoy yet.

![Screenshot 2025-03-07 at 1 29 26 PM](https://github.com/user-attachments/assets/eec7d63a-1f4f-4ad5-b8d1-e603305dbf17)

Reply:
Ah ok you're right, let me remove that target.

Reply:
The target for PATH without GUARD (ie build and run from the binary) is `make path_run` btw

Reply:
OK fixed all refs to point to `make path_run` now.